content/yaml-publishing/distribution.md

```diff
@@ -256,6 +256,8 @@ publishing:
 
 If you wish to pass release notes with your build, create a `release_notes.txt` file and add it to the project working directory, which is either the repository root directory or the Project path specified in the Build section in your workflow settings. Codemagic will fetch the content of that file and publish it with the build.
 
+In order to distribute an `.aab` to testers via Firebase App Distribution, your Firebase project must be linked to your Google Play account. More information is available [here](https://firebase.google.com/docs/app-distribution/android/distribute-console?
+
```

Comment:
the link is incomplete and the sentence is missing a full stop

Reply:
attached linked should be fine now